build/config/phone/custom-prefs.js

```diff
@@ -2,3 +2,4 @@
 user_pref('devtools.responsiveUI.customWidth', 320);
 user_pref('devtools.responsiveUI.customHeight', 480);
 user_pref('devtools.responsiveUI.currentPreset', 'custom');
+user_pref('browser.sessionhistory.max_total_viewers', 3);
```

Comment:
We should not commit this yet, or enable other bug to do this.

We will need to check with Fabrice and Vivien if we are ok with this change, that I hope they are :)